egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -228,7 +228,9 @@ def process_args(args):
         args.transform_dir = args.lat_dir
     # set the options corresponding to args.use_gpu
     run_opts = common_train_lib.RunOpts()
-    if args.use_gpu:
+    if args.use_gpu in ["true", "false"]:
+        args.use_gpu = {'true': 'yes', 'false': 'no'}[args.use_gpu]
```

Comment:
I'd prefer if you wrote this as
 `args.gpu = ("yes" if args.gpu == "true" else "no")`
as it's easier for a human to parse.
Otherwise this is OK.